integrations/pgvector-documentstore.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: integration
+name: pgvector
+description: A Document Store for storing and retrieval from pgvector
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/pgvector-haystack/
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/pgvector
+type: Document Store
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
+version: Haystack 2.0
+toc: true
+---
+
+[![PyPI - Version](https://img.shields.io/pypi/v/pgvector-haystack.svg)](https://pypi.org/project/pgvector-haystack/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pgvector-haystack.svg)](https://pypi.org/project/pgvector-haystack/)
+[![test](https://github.com/deepset-ai/haystack-core-integrations/actions/workflows/pgvector.yml/badge.svg)](https://github.com/deepset-ai/haystack-core-integrations/actions/workflows/pgvector.yml)
+
+-----
+
+**Table of Contents**
+
+- Pgvector Document Store for Haystack
+  - [Installation](#installation)
+  - [Usage](#usage)
+  - [Examples](#examples)
+  - [License](#license)
+
+## Installation
+pgvector is an extension for PostgreSQL that adds support for vector similarity search.
+
+To quickly setup a PostgreSQL database with pgvector, you can use Docker:
+```bash
+docker run -d -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_DB=postgres ankane/pgvector
+```
```

Comment:
thank you for including docker instructions! They made it very easy to get up and running.